web/src/components/molecules/Dashboard/ProjectList.tsx

```diff
@@ -0,0 +1,98 @@
+import { PlusOutlined } from "@ant-design/icons";
+import styled from "@emotion/styled";
+import Button from "@reearth-cms/components/atoms/Button";
+import ProjectCard from "@reearth-cms/components/molecules/Dashboard/ProjectCard";
+import { Project as ProjectType } from "@reearth-cms/components/molecules/Dashboard/types";
+import React from "react";
+
+export interface Props {
+  className?: string;
+  projects?: ProjectType[];
+  workspaceId?: string;
+  handleProjectModalOpen: () => void;
+}
+
+const ProjectList: React.FC<Props> = ({
+  className,
+  projects,
+  workspaceId,
+  handleProjectModalOpen,
+}) => {
+  return (
+    <StyledDashboardBlock className={className}>
+      {projects?.length ? (
+        <Content>
+          {projects.map((project: ProjectType) => (
+            <ProjectCard
+              key={project.id}
+              project={project}
+              workspaceId={workspaceId}
+            ></ProjectCard>
+          ))}
+        </Content>
+      ) : (
+        <EmptyListWrapper>
+          <Title>No Projects Yet</Title>
+          <Suggestion>
+            Create a new project{" "}
+            <Button
+              onClick={handleProjectModalOpen}
+              type="primary"
+              icon={<PlusOutlined />}
+            >
+              New Project
+            </Button>
+          </Suggestion>
+          <Suggestion>
+            Or read <a href="">how to use Re:Earth CMS</a> first
+          </Suggestion>
+        </EmptyListWrapper>
```

Comment:
The spacing here is a little unlike the design.

Reply:
I added margin, for now, vertical centering doesn't seem to work anymore
I'll create a separate issue for it